packages/Python/lldbsuite/test/lang/swift/implementation_only_imports/library_indirect/TestLibraryIndirect.py

```diff
@@ -0,0 +1,70 @@
+# TestLibraryIndirect.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2019 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
+"""
+Test `@_implementationOnly import` behind some indirection in a library used by the main executable
+"""
+import commands
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import os.path
+import time
+import unittest2
+
+class TestLibraryIndirect(TestBase):
+
+    mydir = TestBase.compute_mydir(__file__)
+    
+    @swiftTest
+    def test_implementation_only_import_library(self):
+        """Test `@_implementationOnly import` behind some indirection in a library used by the main executable
+        
+        See the ReadMe.md in the parent directory for more information.
+        """
+        self.build()
+        def cleanup():
+            lldbutil.execute_command("make cleanup")
+        self.addTearDownHook(cleanup)
+        lldbutil.run_to_source_breakpoint(self, "break here", lldb.SBFileSpec("main.swift"))
+
+        self.expect("fr var", substrs=[
```

Comment:
It may be less fragile to use self.frame().FindVariable("container") and then walk the value object tree programmatically instead of expecting the string output to match up perfectly. `TestObjCIvarDiscovery.py`  seems to have an example of doing that. There is also `lldbutil.check_variable()`.

Reply:
I think you commented that last time, too. :-) I don't want that output; I want to see how the system accounts for the missing fields in how it's displayed to the user.